Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -211,7 +195,7 @@ import BraintreeDataCollector
     /// - Warning: This feature is currently in beta and may change or be removed in future releases. Additionally, this feature is currently only supported with client token authorization.
     public func edit(_ request: BTPayPalVaultEditRequest) async throws -> BTPayPalVaultEditResult {
         try await withCheckedThrowingContinuation { continuation in
-            edit(request) { result, error in
+            edit(request: request) { result, error in
```

Comment:
I opted to just change this for now vs going down the route of making everything downstream use async/await since that'd also require changes to how we use ASWeb. Though this is something to consider with how we want to handle async/await across the board as it'll require some larger changes.

Reply:
Good callout - would be nice to create a plan for slowly migrating everything internally to async-await at some point. Maybe after we drop completionHandler syntax in the next major version 😈 👀

Reply:
Agree! We can also slowly migrate things even in a minor version, but will be much easier to do it from the bottom up and migrate the internal/private methods first and work our way up to the public methods. Maybe I will take a look at some of these next open dev...